### DISCUSSION TITLE
Obsoletions in Mondo

### DISCUSSION THREAD

**Discussion by nicolevasilevsky** [monarch-initiative/mondo#2765] [General]
Terms in Mondo may be obsoleted when they are no longer needed. Mondo terms are never deleted, instead are obsoleted, so that any MONDO ID ever created remains in the ontology.

## When is a Mondo term obsoleted?

- **duplicate term**: duplicate terms are merged with another Mondo term
- **obsoleted in source**: for example, OMIM, Orphanet or GARD may retire or obsolete a term. For example, MONDO:0015173 obsolete autoimmune enteropathy type 2
- **is a phenotype and not a disease**: for example, MONDO:0043606 'obsolete pathologic fracture'
- **out of scope: for example**: MONDO:0000834 'obsolete bone deterioration disease' (see #503)

## When is a Mondo term merged?
- If there are two duplicate terms in Mondo that have the same meaning but come from different sources, the terms can be merged. In this case, the obsoleted term should have a 'replaced by' annotation to the term it is merged with.
- Changes in term label or definition that do not alter the meaning of the term do not usually lead to obsoletion. On the other hand, when a term's definition changes meaning, the term should be obsoleted and a new term created instead. In this case, ontology editors should add a tag 'consider: new term ID', or 'replaced by: new term ID'.

## Obsolete terms should have a link to a GitHub Issue

Every term must have a rdfs:seeAlso to a GitHub issue. Where we don't have one for historical reason, we link to this GitHub discussion, where the community is invited to ask for obsoletion reasons in the comments below.

## Notes

Documentation about **_Merging and Obsoleting_** for Mondo editors is [here](https://mondo.readthedocs.io/en/latest/editors-guide/merging-and-obsoleting/).
_Some of this content was adapted from the [Gene Ontology.](http://wiki.geneontology.org/index.php/Principles_for_term_obsoletion)_
See related ticket: https://github.com/monarch-initiative/mondo/issues/2738

**Comment by nicolevasilevsky** [comment]
@matentzn there is not an individual for 'obsolete in source'. Should I add one?

**Reply by nicolevasilevsky** [reply]
see https://github.com/monarch-initiative/mondo/pull/2814

**Reply by matentzn** [reply]
Try it!

**Comment by cmungall** [comment]
Do we really want to use individuals?

**Reply by matentzn** [reply]
I am not sure its even possible in OBO..

**Comment by matentzn** [comment]
I would have said no - if there wasn’t a precedent! I would prefer an AP
solution ala syonym type, or perhaps just some standard iris. Something
anyways we could promote outside of mondo as well? What would you prefer?

**Comment by cmungall** [comment]
standard URIs, no need to commit to ontological type. They could also be
classes. Valid to link to a class in a triple with an annotation.

I would also opt for human-readable URIs

**Comment by sabrinatoro** [comment]
Obsoletion reason: out of scope

According to [this document](https://github.com/information-artifact-ontology/ontology-metadata/blob/master/src/ontology/omo-edit.owl): 

<owl:NamedIndividual rdf:about="http://purl.obolibrary.org/obo/OMO_0001000">
        <rdf:type rdf:resource="http://purl.obolibrary.org/obo/IAO_0000225"/>
        <obo:IAO_0000115 xml:lang="en">The term was added to the ontology on the assumption it was in scope, but it turned out later that it was not.</obo:IAO_0000115>
        <obo:IAO_0000116 xml:lang="en">This obsolesence reason should be used conservatively. **Typical valid examples are: un-necessary grouping classes in disease ontologies, a phenotype term added on the assumption it was a disease**.</obo:IAO_0000116>
        <obo:IAO_0000117 rdf:resource="http://orcid.org/0000-0001-5208-3432"/>
        <obo:IAO_0000233>https://github.com/information-artifact-ontology/ontology-metadata/issues/77</obo:IAO_0000233>
        <obo:IAO_0000234>https://orcid.org/0000-0001-5208-3432</obo:IAO_0000234>
        <rdfs:label>out of scope</rdfs:label>
    </owl:NamedIndividual>
